cmd/atlas/internal/cmdext/cmdext.go

```diff
@@ -296,7 +297,7 @@ func (l EntLoader) MigrateDiff(ctx context.Context, opts *MigrateDiffOptions) er
 	}
 	m, err := entschema.NewMigrate(
 		sql.OpenDB(opts.Dev.Name, opts.Dev.DB),
-		entschema.WithFormatter(migrate.DefaultFormatter),
+		entschema.WithFormatter(dirFormatter(opts.Dir)),
 		entschema.WithGlobalUniqueID(true),
 		entschema.WithDir(opts.Dir),
```

Comment:
I feel like we could have done that inside Ent, right?

Reply:
Yes, and I prefer to do it there. Let me move this logic there and update the Ent in this repo.

Reply:
Sorry no, because of this: 
![image](https://user-images.githubusercontent.com/7413593/219371964-5bb4d248-f660-4945-a28b-dca298c1bb1a.png)